.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+go:
+    - 1.6.x
+    - 1.7.x
+    - 1.8.x
+    - tip
+
+sudo: required
+
+install:
+    - make install.tools
+
+before_script:
+    - export PATH=$HOME/gopath/bin:$PATH
+
+script:
+    - make .gitvalidation
```

Comment:
@heartlock Could you make the change in your PR? Change the gitvalidation base commit to 022b25259e31784d68192cdcd87a82d6c357a1bb